examples/cluster/index.ts

```diff
@@ -24,6 +24,10 @@ const cluster2 = new eks.Cluster(`${projectName}-2`, {
         "audit",
         "authenticator",
     ],
+    vpcCniOptions: {
+        image: "602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.9.0",
+        initImage: "602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init:v1.9.0",
+    },
```

Comment:
Would be nice to ensure the VPC CNI options were actually honored as well. The validation could be added [here](https://github.com/pulumi/pulumi-eks/blob/master/examples/examples_nodejs_test.go#L42).

Reply:
> Would be nice to ensure the VPC CNI options were actually honored as well. The validation could be added [here](https://github.com/pulumi/pulumi-eks/blob/master/examples/examples_nodejs_test.go#L42). You might have to export some additional information, e.g. the spec from index.ts.

Reply:
My typescript/javascript skills are pretty much non-existent. I'm unsure how to export these.
I've tried:
```
export const image2 = cluster2.core.vpcCni.image;
export const initImage2 = cluster2.core.vpcCni.initImage;
```
but I get:
```
    index.ts(36,45): error TS2339: Property 'image' does not exist on type 'VpcCni'.
    index.ts(37,49): error TS2339: Property 'initImage' does not exist on type 'VpcCni'.
```